README.md

```diff
@@ -91,6 +87,13 @@ The `graphqlHTTP` function accepts the following options:
   * **`validationRules`**: Optional additional validation rules queries must
     satisfy in addition to those defined by the GraphQL spec.
 
+  * **`customValidateFn`**: function which will be used to validate instead of default `validate` from 
+    `graphql-js`.
+
+  * **`customFormatErrorFn`**: An optional function which will be used to format any
+    errors produced by fulfilling a GraphQL operation. If no function is
+    provided, GraphQL's default spec-compliant [`formatError`][] function will be used.
```

Comment:
It should mention deprecated `formatError` and say it will be removed in `1.0.0`.

Reply:
Good point. I've added below.